plugin/kubernetes/controller.go

```diff
@@ -109,6 +109,14 @@ func newdnsController(kubeClient kubernetes.Interface, opts dnsControlOpts) *dns
 		zones:            opts.zones,
 		endpointNameMode: opts.endpointNameMode,
 	}
+	add := func(obj interface{}) { dns.updateModifed() }
+	del := func(obj interface{}) { dns.updateModifed() }
+	upd := func(old, new interface{}) { dns.updateModifed() }
```

Comment:
I don't think this is a good idea. Without the endpoint subsets comparison operations, there will be a continuous stream of updates requiring new AXFRs even though there is no actual, relevant change.